doc/installation_instructions/troubleshooting/offline.rst

```diff
@@ -0,0 +1,58 @@
+Airgapped Systems
+-----------------
+
+SmartSim assumes that dependencies can be retrieved via the Internet.  The
+``smart build`` step can be bypassed by transferring the build artifacts from a
+different machine.
+
+.. warning::
+
+    The Redis Source Available License (which licenses RedisAI) prohibits
+    distributing binaries to third-parties. Thus, compiled binaries should not
+    be shared outside of your organization (see `RSAL v2
+    <https://redis.io/legal/rsalv2-agreement/>`_).
+
+
+The easiest way to accomplish this assumes that you have the following
+- A source machine connected to the internet with SmartSim built (referred to as Machine A).
```

Comment:
The format of this is showing up weird in readthedocs -> add a space after line 16 to resolve the issue